inference-engine/tests/functional/plugin/cpu/shared_tests_instances/low_precision_transformations/concat_with_split_transformation.cpp

```diff
@@ -45,13 +45,16 @@ const std::vector<ConcatWithSplitTransformationParam> testValues = {
     }
 };
 
+const std::pair<std::string, std::map<std::string, std::string>> config = {};
```

Comment:
Why do you use `std::pair<std::string, std::map<std::string, std::string>>`? It is not clear.
I suppose using of `std::map<std::string, std::string>` is enough

Reply:
The first elem of the pair is the name of the custom config which will be included in the test name. It's handled in the same way in other already existing cases (subgraph/two_fake_quantize_to_fullyconnected, subgraph/clamp_fq, single_layer/fake_quantize).